components/automate-ui/src/app/modules/policy/details/policy-details.component.scss

```diff
@@ -40,9 +40,12 @@ tbody > .detail-row {
 }
 
 .id-column {
```

Comment:
Each of the "x-details" pages that shows a one-line table of the item needs these changes individually, so you will see this pattern applied in several files.
I only checked the auth-pages, though; are there others...?